package.json

```diff
@@ -30,7 +30,7 @@
     "@popperjs/core": "^2.9.2",
     "@uvue/core": "^0.1.7",
     "@uvue/server": "^0.1.7",
-    "aeternity-fungible-token": "github:aeternity/aeternity-fungible-token#v2.0.0",
+    "aeternity-fungible-token": "git+https://git@github.com/aeternity/aeternity-fungible-token#v2.0.0",

```

Comment:
is `github:` prefix not working in some case?

Reply:
I had some issues with it in installs for some reason, no clue why